src/main/java/de/uniwue/web/io/PageXMLWriter.java

```diff
@@ -163,13 +164,36 @@ public static Document getPageXML(PageAnnotations result, String pageXMLVersion)
 		factory.setNamespaceAware(true);
 
 		try {
-			return factory.newDocumentBuilder().parse(new ByteArrayInputStream(os.toByteArray()));
+			// Add Page Variables to /PcGts/Page
+			// Note: there probably is a more elegant way to do this, but the PageXMLWriter will be rewritten soon anyway
+			System.out.println("Received orientation: "+ result.getOrientation());
+
+			Document doc = factory.newDocumentBuilder().parse(new ByteArrayInputStream(os.toByteArray()));
+
+			return setPageOrientation(doc, result.getOrientation());
 		} catch (SAXException | IOException | ParserConfigurationException e) {
 			e.printStackTrace();
 			return null;
 		}
 	}
 
+	/**
+	 * Writes the Orientation to a PageXML Document
+	 *
+	 * @param doc PageXML Document
+	 * @param orientation orientation angle in degree
+	 * @return modified Document
+	 */
+	private static Document setPageOrientation(Document doc, double orientation) {
+		Element rootElement = doc.getDocumentElement();
```

Comment:
See above:
Content of the function (which should digest `Page page, double orientation`) should be replaced with something like this:
```Java
page.getAttributes().get("orientation").setValue(new DoubleValue(orientation));
```